README.md

```diff
@@ -77,7 +77,7 @@ For full documentation, check out the README.md for each package:
 - [Ledger](packages/ledger/README.md)
 - [Trezor](packages/trezor/README.md)
 - [Keystone](packages/keystone/README.md)
-
+- [D'CENT](packages/dcent/README.md)
```

Comment:
Will need to add a README for D'Cent describing usage and init options, similar to the format of other package's READMEs

Reply:
@Adamj1232 
I added "README". Please review again.

Reply:
Perfect, thank you!!